plugins/generic/customBlockManager/CustomBlockPlugin.inc.php

```diff
@@ -176,13 +183,29 @@ function manage($verb, $args) {
 	 */
 	function getContents(&$templateMgr) {
 		$journal =& Request::getJournal();
+		$journalId = $journal->getId();
+		
 		if (!$journal) return '';
+		
+		$locale = AppLocale::getLocale();
+		$primaryLocale = AppLocale::getPrimaryLocale();
+		$blockContent = $this->getSetting($journalId, 'blockContent');
+		$blockContentTxt = '';
```

Comment:
Ah, actually, I see where it's used. Mind doing a little bit of cleanup around variable names? It's easy to misread at the moment.